docs/table.rst

```diff
@@ -495,20 +504,118 @@ is set. The above example can therefore be shortened::
 
     $table->addTotals(['client' => 'Totals for {$clients} client(s)']);
 
+In addition to the plans you define, there is also going to be `{$_row_count}` which automatically counts
+number of rows in your table.
+
 Possible values for total plan stages
 -------------------------------------
 
 `init` value is typically a number, but can be any value, especially if you are going to control it's
 increment / formatting.
 
-`update` can be string - either "sum" or "increment". Other values are not supported. You can also pass
-a callable which gives you an option to perform update yourself.
+`update` can be string. Supported built-ins are:
+
+ - min
+ - inc
+ - max
+ - sum
+ - avg
+
+You can use a callable which gives you an option to perform update yourself. Also, make note that `inc`
+will ONLY increment when value of the column it's associated with is not empty. If you need to get
```

Comment:
please be more specific what "empty" means here. Is 0 empty? Is empty string empty? Or is it just null. I think that 0 is not _empty_, but empty string is _empty_.